README.md

```diff
@@ -14,62 +14,33 @@ certification.  Please see "CNF Developers" below for more information.
  ![overview](docs/images/overview.svg)
 
 In the diagram above:
-- the `CNF under test` is the CNF to be certified. The certification suite identifies the pods belonging to the CNF via a label, see [Pod Roles](#pod_roles)
+- the `CNF under test` is the CNF to be certified. The certification suite identifies the resources (containers/pods/operators etc) belonging to the CNF via labels or static data entries in the config file
 - the `Certification container/exec` is the certification test suite running on the platform or in a container. The executable verifies the CNF under test configuration and its interactions with openshift 
-- the `Partner pod` can be any pod with the required tools (e.g. only ping at the moment) in the same namespace as the `CNF under test`. During connectivity tests, the partner pod will generate pings towards the `CNF under test` to verify connectivity.
-
-These roles are configured in the [Test Configuration](#test-configuration) section below.
+- the `Partner pod` can be any pod with the required tools in the same namespace as the `CNF under test`. For example, during connectivity tests, the partner pod will generate pings towards the `CNF under test` to verify connectivity. The partner pods/containers are auto deployed by the test suite prior a test run and can be auto discovered by the suite without any data entry in the config file.
 
 
 ## Test Configuration
```

Comment:
## Test Configuration

The Test Network Function support auto-configuration using labels and annotations, but also a static configuration using a file. 
The following section describe how to configure the TNF via labels/annotation and the corresponding settings in the config file. A sample config file can be found [here](test-network-function/tnf_config.yml).

It consists of the following sections:
When labelling a pod to be discovered and tested, the discoered pods are **in addition to** the ones
explicitly configured in the testTarget sections of the config file.
### targetPodLabels
The goal of this section is to specify the label to be used to identify the cnf under test pods. So for example, with the default configuration:
```shell script
targetPodLabels:
  - namespace: test-network-function.com
    name: generic
    value: target
```

The corresponding label prefix is: 
```shell script
test-network-function.com/generic: target 
```


### testTarget